recipes/mypy-boto3-secretsmanager/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "mypy-boto3-secretsmanager" %}
+{% set version = "1.26.89" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mypy-boto3-secretsmanager-{{ version }}.tar.gz
+  sha256: 4e08ec1ae81229fb20e16af40004923f86f6fe119fa7730edb289cd851fa9e55
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - typing-extensions >=4.1.0
```

Comment:
You need a dependency on `botocore`. I'm not sure why the upstream `setup.py` doesn't have it, but the package clearly imports it.

Reply:
Aha, that's where to add it. Thank you!